tsconfig.json

```diff
@@ -1,22 +1,20 @@
 {
   "compilerOptions": {
-    "target": "ES2015",
-    "module": "ES2015",
-    "lib": ["es5", "es2015", "dom"],
+    "target": "ES5",
+    "module": "commonjs",
```

Comment:
Currently we build according to `ES5` rules.
It is created in `commonjs` format for use in general `js`.